src/NuGet.Services.AzureSearch/SearchService/Models/ApiType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Services.AzureSearch.SearchService
+{
+    public enum ApiType
+    {
+        /// <summary>
+        /// The data for the user was fetched using the "get document by key" API. The .NET API is called "Get" and
```

Comment:
It's not super clear you're referring to Azure Search until the link. How about something like:

```suggestion
        /// The data for the user was fetched using Azure Search's "get document by key" API. The .NET API is called "Get" and
```

Same comment for `Search`'s doc